docs/reference/tab-widgets/inference-api/infer-api-task.asciidoc

```diff
@@ -0,0 +1,55 @@
+// tag::cohere[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/cohere_embeddings <1>
+{
+    "service": "cohere",
+    "service_settings": {
+        "api_key": "<api_key>", <2>
+        "model": "embed-english-light-v3.0" <3>
+    },
+    "task_settings": {
+    }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The task type is `text_embedding` in the path.
+<2> The API key of your Cohere account. You can find your API keys in your
+Cohere dashboard under the
+https://dashboard.cohere.com/api-keys[API keys section]. You need to provide
+your API key only once. The <<get-inference-api>> does not return your API
+key.
+<3> The name of the embedding model to use. You can find the list of Cohere
+embedding models https://docs.cohere.com/reference/embed[here].
+
+// end::cohere[]
+
+
+// tag::openai[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/openai_embeddings <1>
+{
+    "service": "openai",
+    "service_settings": {
+        "api_key": "<api_key>" <2>
+    },
+    "task_settings": {
+       "model": "text-embedding-ada-002" <3>
```

Comment:
Unrelated to the cohere stuff but we decided to move this to the `service_settings`. We also renamed it to `model_id`. So `"model_id": "text-embedding-ada-002"`